packages/firebase_analytics/android/src/main/java/io/flutter/plugins/firebaseanalytics/FirebaseAnalyticsPlugin.java

```diff
@@ -62,6 +103,10 @@ public void onMethodCall(MethodCall call, Result result) {
     }
   }
 
+  private void setActivity(Activity activity) {
+    this.activity = activity;
```

Comment:
Where is this activity being used?

Reply:
There is a event that logs the current screen `handleSetCurrentScreen`, the native SDK requires you pass in the current activity.